cypress/integration/dropdown.js

```diff
@@ -47,7 +47,16 @@ describe('Dropdown test', function() {
                 .get('h4').contains('GitLab').contains('Link Account')
         });
     });
-
+    var everythingOk = function() {
+        cy
+        .get('h4').contains('GitHub').contains('Unlink Account')
```

Comment:
Is there a way to use IDs rather than just headers that contain strings? 
This seems a bit fragile